aten/src/ATen/native/ConvUtils.h

```diff
@@ -78,7 +78,11 @@ namespace {
 }
 
 inline bool cudnnv8_enabled_check_debug() {
-  static bool cudnnv8_flag = c10::utils::check_env("TORCH_CUDNN_V8_API_DISABLED") != true;
+  auto cudnnv8_flag_opt = c10::utils::check_env("TORCH_CUDNN_V8_API_DISABLED");  
```

Comment:
Ugh, should it still be static though @cyyever

Reply:
Should keep the static behavior